README.md

```diff
@@ -102,9 +101,25 @@ See [BUILD.md](BUILD.md)
 
 See http://www.konstructs.org/download/
 
+### Connecting to a Server
+
+There are multiple ways to connect to a server. The most common way is to just start the game and follow the on-screen prompts.
+
+A more advanced way is through commandline arguments. The advantage of this is, that commandline arguments can be included in link files (on Windows .lnk, on Linux sometimes calls "Starter") to automatically connect to your favorite server!
```

Comment:
Er, if I've heard about starter? Or did you never hear about it, how should I understand this sentence? :D
I only know that it is called "starter" on german debian-based linux, but I'm not sure whether it's the common word for it?